api/src/endpoints/repository.js

```diff
@@ -41,4 +41,6 @@ routes.get('/models', async (req, res) => {
   }
 });
 
+routes.use('/models', express.static('repository'));
+
```

Comment:
How about adding
```
app.use('/models', express.static('repository'));
```
to [index.js](https://github.com/MetabolicAtlas/MetabolicAtlas/blob/develop/api/src/index.js)?

Reply:
I see that argument and I had it there first actually. But I wanted to follow the old URL pattern as well as the route nesting architecture we have set up in the `api` so it made a bit more sense for me to have in the `repository.js` file.